### ISSUE TITLE
Release to playstore

### ISSUE THREAD

**Issue by n-ce** [qw123wh/Aospdeskclock#15]
Faster & Easier User Side Updates and also being able to publish split apk for reduced file size.

**Comment by qw123wh** [comment]
switch to this other updated version
https://github.com/qw123wh/new-clock/releases/tag/1.0
however the app weighs only 5mb, there is no need to split it
for the play store I don't have a developer account, if someone wants to publish it for me, that's no problem for me

**Comment by moriel5** [comment]
@qh123wh Great job with the AOSP clock.

Do you think you could merge the UI in this certain with the newer version? That could theoretically as more option to for now people's tastes (new UI with older colors, new UI with newer colors, older UI with older colors, older UI with newer colors), as well as open up the path for some really nice features (the old and unfortunately removed feature where the colors of the app changes in accordance to what time it is is one such useful feature).

**Comment by qw123wh** [comment]
@moriel5 
In the end the change between the two apps is only aesthetic, the functionalities are the same, here 4 functions are missing compared to the new one:

1) automatic light and dark theme;
2) swipe to delete alarm; 
3) function of the power button to turn off / postpone the alarm; 
4)new analog clock.  

It is very difficult and tiring to create a single app born from the union of these two.  I prefer to have separate projects in order to satisfy both those who prefer the old interface and those like me who prefer the new interface.  The names of the two apps are different, so they can both be installed on the same device at the same time.

If you have any requests, create a new issue and then we will see if I succeed or if someone else will be able to implement it, if you know any interested developer or you want to contribute yourself, you can do it with pull requests.  I am looking for collaborators for these apps.

**Comment by moriel5** [comment]
@qw123wh That is true (and I am more of a fan of the old UI), I had thought that it might actually be easier on you in the long term if you unify them.

Regarding the missing functions, these are all things I would love to see in the old UI, preferably together with the return of the deprecated feature I had previously mentioned (the colors of the app change in accordance to what part of the day it is).

**Comment by qw123wh** [comment]
@moriel5 It can be done on function n2 and n4.  On function n3 there are some crashes in the app that I can't solve.  On function n1 it will not be implemented (too much work, bugs and various problems).

As for the deprecated function you say, I need an old app to understand the function (if you have any old version with this function send it to me, or maybe a video of how it works if you can't find the app, if you then find as well commits about this function in the old aosp clock source codes, then we have some chance.

for ease if you want you can also write to me by email

qw123whreuvkjs@gmail.com

**Comment by moriel5** [comment]
Understood, what can be done can be done, and what cannot, cannot.

Unfortunately, I lack the development skills to be able to work on the code itself, otherwise I would have tried to fix this myself, however I'm willing to do whatever I can do to help.

Regarding that old feature, I'll look into it, as Android Police has reported it's removal from the proprietary Google Clock app, however as it was also in the AOSP sources at the time, I should be able to figure out which version that was (I believe around the time of Android 5.0-5.1, but I'll check).

Via email, is that preferable to you? As I am often on a device with bugs in the WiFi driver (this is temporary, until I have enough time to replace the ROM) and as such, some apps simply fail to upload anything beyond a few kilobytes (like most email apps), so it is currently easier for me via GitHub (I use FastHub).

**Comment by qw123wh** [comment]
send everything by email is more convenient.

**Comment by moriel5** [comment]
@qw123wh Thanks, I'll get to that once I have enough time on my laptop or desktop.
